.gitignore

```diff
@@ -1,3 +1,19 @@
 /target
 /vendor/boost
 /touchHLE_sandbox
+
+# Android
+
+*.iml
+.gradle
+/local.properties
+.idea/
+.DS_Store
```

Comment:
`.DS_Store` isn't Android-specific. I recommend using a global git “excludes file” so you don't have to put this in a `.gitignore` file in every repository.

Reply:
Done